docs/content/en/docs/implementing/slo/_index.md

```diff
@@ -0,0 +1,122 @@
+to migrate towards Keptn Analysis.
+
+## Keptn Analysis basics
+
+A Keptn Analysis is implemented with three resources:
+
+* [Analysis](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysis) --
+  define the specific configurations and the analysis to report
+* [AnalysisDefinition](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysisdefinition) --
+  define the list of SLOs for an analysis
+* [AnalysisValueTemplate](https://lifecycle.keptn.sh/docs/crd-ref/metrics/v1alpha3/#analysisvaluetemplate) --
+  define the SLI: the KeptnMetricsProvider and the Query to perform for each SLI
+
+### Define Analysis, Analysis Definition and AnalysisValueTemplate
+
+An Analysis customizes the templates defined inside an AnalysisDefinition by adding configurations such as:
+
+* a Timeframe that specifies the range for the corresponding query in the AnalysisValueTemplate
+* a map of key/value pairs that can be used to substitute placeholders in the AnalysisValueTemplate
+
+An AnalysisDefinition contains a list of objectives to satisfy.
+Each of these objectives:
+
+* specifies failure or warning target criteria,
+* specifies whether the objective is a key objective (its failure would fail the analysis)
+* indicates the weight of the objective on the overall analysis
+* refers to an AnalysisValueTemplate that contains the SLIs, defining the data provider from which to gather the data
+  and how to compute the analysis
+
+In each AnalysisValueTemplate we store the query for the analysis of the SLI.
+You must define a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md) resource
+for each instance of each data provider you are using.
+The template refers to that provider and queries it.
+
+Let's consider the following Analysis:
+
+{{< embed path="/metrics-operator/config/samples/metrics_v1alpha3_analysis.yaml" >}}
+
+This CR sets up the timeframe we are interested in
+as between 5 am and 10 am on the 5th of May 2023,
+and adds a few specific key-value pairs that will be substituted in the query.
+For instance, the query could contain a `{{.nodename }}` and this value will be substituted by test
+
+The definition of this Analysis is referenced by its name and namespace and can be seen here:
+
+{{< embed path="/metrics-operator/config/samples/metrics_v1alpha3_analysisdefinition.yaml" >}}
+
+This simple definition contains a single objective, `response-time-p95`.
+For this objective, there are both
+failure and warning criteria:
+
+* objective will fail if the percentile 95 is less than 600
+* there will be a warning in case the value is in between 300 and 500
+
+The total score shows that this analysis should overall score 90% of all objectives to pass or 75 to get a warning.
+Since the objective is one only, this means that we either will pass with 100% (response time is less than 600) or fail
+with 0%(slower response time)
```

Comment:
```suggestion
The total score shows that this analysis should have an overall score of 90% to pass or 75% to get a warning.
Since the objective is only one, this means that we either will pass with 100% (response time is less than 600) or fail
with 0% (slower response time).
```